reading/supervisors.livemd

```diff
@@ -133,17 +133,9 @@ Every supervisor is also a process. The [Supervisor.start_link/2](https://hexdoc
 
 ```elixir
 children = [
-  %{
-    id: :worker1,
-    start: {Worker, :start_link, [1]}
-  },
-  %{
-    id: :worker2,
-    start: {Worker, :start_link, [2]}
-  },
-  %{
-    id: :worker3,
-    start: {Worker, :start_link, [3]}
+  {
+    Domino,
+    [1]
```

Comment:
To be idiomatic, you can default opts as an empty list so you don't need the `1`.

Reply:
Also, is this related to this PR? I don't think so, so unless I'm wrong, let's remove it.